### PR TITLE
Add new bluespace storage items

### PR DESCRIPTION
**Pull request by hyperioo**
## About The Pull Request

Part of the RnD rework. These new items can be unlocked in the Bluespace tree of RnD with a new node.
Adding belt of holding, pouch of holding, trashbag of holding and satchel of holding (infinite ores and hydroponics products storage).

As the bluespace backpack, these items interfere with teleportation and with the singularity.

* Thanks to INFRARED_BARON for all the new bluespace sprites

* To be merged after #5134 since it uses a modified `/nano/images/science.png` for the icon of the research node

## Why It's Good For The Game

More late-game storage items and something new to unlock for Moebius.

## Changelog
:cl: Hyperio
add: Added new bluespace storage items
/:cl:

### CODE REVIEW

**Comment by GrayRachnid** [comment]
will these affect teleportation like the bag of holding?
Should they?

**Comment by hyperioo** [comment]
No, they're just upgraded version of their standard counterpart (huge storage).
I don't know if they should, as you want.

File: code/game/objects/items/weapons/storage/bags.dm [hunk 1]
```diff
@@ -97,6 +113,20 @@
 	max_w_class = ITEM_SIZE_NORMAL
 	can_hold = list(/obj/item/weapon/ore)
 
+/obj/item/weapon/storage/bag/ore/holding
+	name = "satchel of holding"
+	desc = "A revolution in convenience, this satchel allows for infinite ore or produce storage. It's been outfitted with anti-malfunction safety measures."
+	icon_state = "satchel_bspace"
+	max_storage_space = INFINITY
```
Comment (Kurgis) [inline code/game/objects/items/weapons/storage/bags.dm:120]: Sadly there's a maximum cap of what BYOND can allow, I need to find out what it actually is though.

Comment (hyperioo) [reply code/game/objects/items/weapons/storage/bags.dm:120]: @Kurgis I can put a big finite value, just to be safe. I actually took that from Paradise code so I guess if their BYOND can handle `INFINITY` then ours should also be able to do it right?

Comment (Kurgis) [reply code/game/objects/items/weapons/storage/bags.dm:120]: In that case it may of been a legacy issue, which no longer occurs.

Comment (Fikou) [reply code/game/objects/items/weapons/storage/bags.dm:120]: infinity is just a very high nubmer in byond

**Comment by SpadesSlick** [comment]
Creating one should add some Bluespace entropy introduced in #5257, this probably can be done in ``Init`` without much problem. Both PRs are not merged tho, yet.

**Comment by SpadesSlick** [comment]
> I don't know if they should, as you want.

They should, yeah.

**Comment by SpadesSlick** [comment]
``Holdingpouch`` currently is the best sprite in whole selection, all other sprites must be redone to match it's style.

**Pull request merged**